app/quizzes/[id]/choice-form.tsx

```diff
@@ -1,22 +1,17 @@
 'use client';
 
 import React, { useState } from 'react';
-import { useGetChoicesOfQuiz, postQuizSubmission } from '@/hooks/quiz';
-import { useMutation } from '@tanstack/react-query';
+import { useGetChoicesOfQuiz, useSubmitQuiz } from '@/services/quiz/hooks';
 import { useRouter } from 'next/navigation';
 import Button from '@/components/common/buttons/button';
 import LoadingSpinner from '@/components/common/loading-spinner/loading-spinner';
 
 export default function ChoiceForm({ quizId }: { quizId: number }) {
-  const { data: choices } = useGetChoicesOfQuiz(quizId);
   const [errorMessage, setErrorMessage] = useState('');
   const router = useRouter();
 
-  const { mutate, isPending, isSuccess } = useMutation({
-    mutationFn: postQuizSubmission,
-    onSuccess: () => router.push(`/quizzes/${quizId}/answer`),
-    onError: (error) => setErrorMessage(error.message),
-  });
+  const { data: choices } = useGetChoicesOfQuiz(quizId);
+  const { mutate, isPending, isSuccess } = useSubmitQuiz();
```

Comment:
저는 `mutate` 이름도 `submitQuiz`로 바꾸면 더 직관적일 것 같다는 생각이 들었네요..!
`use+동작`에서 `use`만 제거해서 네이밍하면 통일되게 작성할 수 있을 것 같아요! 🙇‍♂️

```suggestion
  const { mutate: submitQuiz, isPending, isSuccess } = useSubmitQuiz();
```